app/assets/stylesheets/application.css

```diff
@@ -156,3 +156,8 @@ td, th {
 tr:nth-child(even) {
   background-color: #F0F8FF;
 }
+
+.item-image {
+  width: 150px;
+  height: 100px;
+}
```

Comment:
Sized images so in the order show view page the images aren't huge

Reply:
This looks much better.